extension/src/projectBuilder.ts

```diff
@@ -13,18 +13,43 @@ export const build = (terminal: vscode.Terminal): void => {
         return;
     }
     const isWindowsPlatform = /^win/.test(process.platform);
+    const mono = (isWindowsPlatform) ? "" : "mono";
+
+    let buildCommand = `${xcbuildPath} --compilationmode=Debug --build --env=Dev --vs=VS2015 --project=${xcmlPath}`;
+    let exportXCRCommand = `${xcbuildPath} --exportRuntimes --compilationmode=Debug --env=Dev --output="${rootPath}${path.sep}XCR" --project=${xcmlPath}`;
```

Comment:
I prefer not to create terminology here....

just call them `exportRuntimesCommand` and `exportInterfaceCommand` and it will be clearer for everybody.